pages/_app.js

```diff
@@ -1,10 +1,27 @@
 import GlobalStyles from "../components/GlobalStyles";
+import { useLocalStorage } from "../helpers/hooks";
 
 function MyApp({ Component, pageProps }) {
+  const [morningAnswers, setMorningAnswers] = useLocalStorage(
+    "morningAnswers",
+    []
+  );
+
+  function addMorningAnswer(answer) {
+    setMorningAnswers((previousMorningAnswers) => [
+      ...previousMorningAnswers,
+      { id: crypto.randomUUID(), text: answer },
```

Comment:
Where is `crypto.randomUUID` coming from?

Reply:
This is actually coming from the [crypto API (browser API)](url)

Reply:
Hej Martin, that was something I added together with my coaches when coding the localStorage. It's generating an id within the browser. Does that answer your question? :)

Reply:
We had this before in one of our sessions already

Reply:
Thanks for the info.